packages/flutter/lib/src/painting/rounded_rectangle_border.dart

```diff
@@ -211,7 +211,10 @@ class _RoundedRectangleToCircleBorder extends _ShapeToCircleBorder<RoundedRectan
 ///  * [RoundedRectangleBorder], which uses the traditional [RRect] shape.
 class RoundedSuperellipseBorder extends OutlinedBorder with _RRectLikeBorder {
   /// Creates a rounded rectangle border.
-  const RoundedSuperellipseBorder({super.side, this.borderRadius = BorderRadius.zero});
+  ///
+  /// The `borderRadius` defaults to [BorderRadius.zero].
```

Comment:
Nit: "The `borderRadius` defaults to [BorderRadius.zero] if `borderRadius` is null." to clarify that it defaults both when no value is provided, and if a value of null is provided.